### ISSUE TITLE
Decide on official mascot

### ISSUE THREAD

**Issue by petejohanson** [zmkfirmware/zmk#195]
Placeholder issue until I can add all the suggested options for folks to give feedback.

Needed before we can make stickers!

**Comment by procom32** [comment]
Zeplin Zebras :)
https://i.imgur.com/V4gY3fy.jpg

**Comment by Na-Cly** [comment]
Normally I can only stand to 3 stock fox, but in this case I think Zorro.

**Comment by petejohanson** [comment]
![Screenshot_20200929-135800.png](https://user-images.githubusercontent.com/202695/94597642-d2644c00-025b-11eb-8be0-927514a32164.png)

The Zapata Wren seems to be a fairly strong contender, with the Zorro not that far behind. Will consider and make a final decision on this soon. Once that's done, we'll need to find a designer to help out!

**Comment by petejohanson** [comment]
Ok, so slept on this for a day. 

Zapata Wren it is! They're super cute, and I think will translate well to both a full mascot, and smaller logo formats.

I'll be closing this issue shortly, and open a new issue to track trying to get the new graphics created.

Thanks everyone!
